node/cn/tracers/tracer.go

```diff
@@ -158,14 +159,14 @@ type stackWrapper struct {
 }
 
 // peek returns the nth-from-the-top element of the stack.
-func (sw *stackWrapper) peek(idx int) *big.Int {
+func (sw *stackWrapper) peek(idx int) *uint256.Int {
```

Comment:
The type of return value doesn't need to be changed since anyway it will be converted into big.Int. What about maintaining the original type?

Reply:
Most callers aren't converting to bigint. Can you specify why? Could you pinpoint regarding this?

Reply:
I mean this `peek` function. This function is only called in one place, this line. And it requires big.Int so I thought it's okay to keep the original return value type.
https://github.com/klaytn/klaytn/blob/4f572c3b8bd5aaedac7187dbc946388d5118946f/node/cn/tracers/tracer.go#L184

Reply:
I'm sorry. I thought you pointed the `stack`'s `peek` function. Now it reverted, but still, there's another update because the `stack.Data()` returns `uint256`.